src/hotspot/share/classfile/classFileStream.hpp

```diff
@@ -52,12 +52,12 @@ class ClassFileStream: public ResourceObj {
   const char* clone_source() const;
 
  public:
-  static const bool verify;
+  constexpr static bool verify = true;
```

Comment:
Where, and how, is this used?

Reply:
It's passed into the ClassFileStream constructor with default value true.

Reply:
I removed this and fixed all the constructor calls.